lib/MusicBrainz/Server.pm

```diff
@@ -261,7 +261,7 @@ sub get_collator
 
 sub set_language_cookie {
     my ($c, $lang) = @_;
-    $c->res->cookies->{lang} = { 'value' => $lang, 'path' => '/', 'expires' => time()+31536000 };
+    $c->res->cookies->{lang} = { 'value' => $lang, 'path' => '/', 'expires' => time()+31536000, 'SameSite' => 'None', 'Secure' => '1' };
```

Comment:
This didn't work for me, but this does:
```suggestion
    $c->res->cookies->{lang} = {
        'value' => $lang,
        'path' => '/',
        'expires' => time()+31536000,
        $c->req->secure ? (
            'samesite' => 'None',
            'secure' => '1',
        ) : (),
    };
```
 * samesite/secure have to be lowercase here (dunno why)
 * they should only be output if `$c->req->secure`, otherwise the browser will reject the cookie on dev servers where SSL is unlikely to be set up
 * added some indentation for clarity